.github/workflows/add_milestone.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       GH_REPO: ${{ github.repository }}
     steps:
       - name: Checkout datadog-agent repository
-        uses: actions/checkout@v4
+        uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
```

Comment:
Would it make sense to have a centralized file where these `version->hash` maps are kept? Maybe we can also inject those as values into the build scripts? Without something like that, verification of proper hash value use becomes extremely had to maintain and review.

Reply:
I'll check if we can do this. It could remove some redundancy even if the updates are normally handled automatically by dependabot (so mitigates a little bit the maintenance complexity, wdyt?)

Reply:
I really disagree with the single file approach, we know in advance it will never be maintained and upgrades will never be made. Keeping the format easy for dependabot to do the upgrades on its own should be the main priority IMO.

Reply:
We still have the associated tag in the comment for human-readable format. Would you prefer having a link to the release on the corresponding github repo?

Reply:
@chouetz That may be a good compromise though a link to the repo at that tag release would be better I think since the hash can easily be compared.